library/pom.xml

```diff
@@ -378,6 +378,11 @@
         </exclusion>
       </exclusions> 
     </dependency>
+      <dependency>
+          <groupId>org.apache.hadoop</groupId>
+          <artifactId>hadoop-hdfs</artifactId>
+         <version>2.6.1</version>
+      </dependency>
```

Comment:
Apex-archetype skips the hadoop-hdfs jar in app package, I have tested that. I will explore provided scope.